src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -1,34 +1,49 @@
-import { NgModule } from '@angular/core';
-import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+/* eslint-disable arrow-body-style */
+import { HttpClient } from '@angular/common/http';
+import { NgModule, isDevMode } from '@angular/core';
+import { AuthModule, LogLevel, StsConfigHttpLoader, StsConfigLoader } from 'angular-auth-oidc-client';
+import { map } from 'rxjs/operators';
+import { environment } from 'environments/environment';
 
+export const httpLoaderFactory = (httpClient: HttpClient) => {
+  const config$ = httpClient.get<any>(`assets/oidc.json`).pipe(
+    map((customConfig: any) => {
+      return {
+        triggerAuthorizationResultEvent: true,
+        postLoginRoute: window.location.origin + customConfig.postLoginRoute,
+        logLevel: environment.production ? LogLevel.None : LogLevel.Debug, // or can use isDevMode() ?
+        //historyCleanupOff: true, //commented now, will come back later
```

Comment:
Hey, On top of my head I am not sure what the default value is. Is it False by default ? If not then 
how about for now setting it to historyCleanupOff: false  ?

Reply:
please see previous reviewed items, I mentioned that this line  //historyCleanupOff: true, //commented now, will come back later was deleted.